optiboot/bootloaders/optiboot/pin_defs.h

```diff
@@ -1628,3 +1628,26 @@
 #if SOFT_UART && defined(SOFTTX) && defined(SOFTRX)
 #include "pins_softuart.h"
 #endif
+
+#ifdef HOLD_TO_PROG
+
+#if HOLD_TO_PROG == D0
+#undef HOLD_TO_PROG
+#define HOLD_TO_PROG_PIN     PIND
+#define HOLD_TO_PROG	    PIND0
+
+#else
+#error -------------------------------------------
+#error Unrecognized LED name.  Should be like "B5"
```

Comment:
I think this is incorrect wording probably an effect of cut&paste. Can you re-word please?

Reply:
Yeah, I'll change it (and yeah, it's cut and paste).